Add snmp Cisco tile (#8156)
DataDog/integrations-core: snmp_cisco/README.md

# Agent Check: snmp_cisco

## Overview

Cisco is the worldwide leader in IT, networking, and cybersecurity solutions. Install the integration to monitor all of your Cisco appliances, including routers, switches, voice gear, security appliances and more.

Collect SNMP metrics from Cisco appliances, including:

- Cisco Catalyst
- Cisco Adaptive Security Appliance (ASA)
- Cisco Meraki (note: additional events can be collected from Meraki through the [Meraki integration tile][1])
- Cisco Nexus
- Cisco ICM
- Cisco ISR

> - Cisco UC Virtual Machines

Reviewer: My vote is for no, only because we would be linking to the github repo, and on occasion that is out of sync with what's been officially shipped/released with the agent, and have seen customers complain that what they're seeing does not match what's in the repo. By not linking to the source code we give ourselves a little more room to control what customers expect

## Setup

> To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.

Author: Do you mean listing all devices supported? The list can be quite long

> To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.

Author: Should it be here, or in the `snmp` tile?

> 

Reviewer 2: This readme is missing the Metrics, events, and service check sections, is that intentional?

Reviewer 3: Yes, because there is none. 

We could also mention it:

```
### Service Checks

snmp-cisco does not include any service checks.

### Events

snmp-cisco does not include any events.
```

# Agent Check: snmp_cisco

## Overview

Cisco is the worldwide leader in IT, networking, and cybersecurity solutions. Install the integration to monitor all of your Cisco appliances, including routers, switches, voice gear, security appliances and more.

Collect SNMP metrics from Cisco appliances, including:

- Cisco Catalyst
- Cisco Adaptive Security Appliance (ASA)
- Cisco Meraki (note: additional events can be collected from Meraki through the [Meraki integration tile][1])
- Cisco Nexus
- Cisco ICM
- Cisco ISR
- Cisco UC Virtual Machines

> For details of monitored metrics see the [SNMP integration tile][2]

Reviewer 4: ```suggestion
For details on monitored metrics see the [Network Device Monitoring][2] documentation.
```

## Setup

To install and configure the SNMP integration, see the [Network Device Monitoring][3] documentation.

## Data Collected

### Metrics

> For details of monitored metrics see the [SNMP integration tile][2]

Reviewer 4: ```suggestion
For details on monitored metrics see the [Network Device Monitoring][2] documentation.
```

### Service Checks

> Snmp Cisco does not include any service checks.

Reviewer 4: ```suggestion
The SNMP Cisco integration does not include any service checks.
```

### Events

> Snmp Cisco does not include any events.

Reviewer 4: ```suggestion
The SNMP Cisco integration does not include any events.
```

## Further Reading

Additional helpful documentation, links, and articles:

* [Monitor SNMP with Datadog][4]

[1]: https://app.datadoghq.com/account/settings#integrations/meraki

> [2]: https://app.datadoghq.com/account/settings#integrations/snmp

Reviewer 4: ```suggestion
[2]: https://docs.datadoghq.com/network_performance_monitoring/devices/data
```
